libc/test/src/math/CMakeLists.txt

```
@@ -202,6 +202,20 @@ add_fp_unittest(
     libc.src.__support.FPUtil.basic_operations
 )
 
+add_fp_unittest(
+  faddl_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    faddl_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.faddl
+    libc.src.__support.FPUtil.basic_operations
+)
```

libc/test/src/math/AddTest.h doesn't include BasicOperations.h.

```suggestion
  DEPENDS
    libc.src.math.faddl
)
```